Sprint-1/interpret/to-pounds.js

```diff
@@ -25,3 +25,15 @@ console.log(`£${pounds}.${pence}`);
 
 // To begin, we can start with
 // 1. const penceString = "399p": initialises a string variable with the value "399p"
+
+ // 2.const penceStringWithoutTrailingP = penceString.substring( 0, penceString.length - 1);:it initializing the string from the start(index[0]) by excluding the last character therefore,the p will be excluded//399
+
+ //3.const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"): padStart makes sure that paddedPenceNumberString is three characters long//399
+
+ //4.const pounds = paddedPenceNumberString.substring(  0, paddedPenceNumberString.length - 2):it initializing the string from the start(index[0]) by excluding the last two characters//3
+
+ //5.const pence = paddedPenceNumberString.substring(paddedPenceNumberString.length - 2) .padEnd(2, "0"):two functions here 
+
+   //the first one paddedPenceNumberString.substring(paddedPenceNumberString.length - 2) : This calculates the starting index for the substring, which is two characters from the end of the string//99
+   //the second one is padEnd(2, "0") which takes the last two characters of the string//99
```

Comment:
Can you look up what `.padEnd()` does? 
Also, can we remove `.padEnd(2, "0")` from the code and expect the code to work correctly as before?